src/lang.rs

```diff
@@ -127,7 +127,33 @@ fn consume_string(pairs: &mut Pairs<Rule>) -> Option<String> {
   pairs
     .next()
     .and_then(|x| x.into_inner().next())
-    .map(|x| x.as_str().to_string())
+    .map(|x| unescape(x.as_str()))
+}
+
+/// Given a &str, unescape special characters
+///
+/// This should potentially return an error, but that really complicates the
+/// above API, so it just silently permits and ignores invalid escapes. Oops.
```

Comment:
:fearful:

Reply:
Yeah. I'll reach into fixing this up later, but for now I'm letting it slide.